webserver/views/datasets.py

```diff
@@ -144,7 +147,11 @@ def edit(dataset_id):
     ds = get_dataset(dataset_id)
     if ds["author"] != current_user.id:
         raise Unauthorized("You can't edit this dataset.")
-
+    jobs = db.dataset_eval.get_jobs_for_dataset(str(dataset_id))
+    for job in jobs:
+        if(job["status"]=="running" or job["status"]=="pending"):
+            flash.warn("There is a pending job for this dataset")
```

Comment:
Dataset pages are using React, so it's probably a matter of inserting more values into one of json endpoints.